Config.cmake/runtime_test.cmake

```diff
@@ -82,29 +83,22 @@ function(to_native_c_path path native_c_path)
     set (${native_c_path} ${tmp} PARENT_SCOPE)
 endfunction()
 
-set(UNCOMPRESSED_DUMMY_FILE_PATH_SRC "${FMIL_TEST_DIR}/try_to_uncompress_this_file.zip")
-set(UNCOMPRESSED_DUMMY_FOLDER_PATH_DIST "${TEST_OUTPUT_FOLDER}")
-file(COPY "${UNCOMPRESSED_DUMMY_FILE_PATH_SRC}" DESTINATION "${UNCOMPRESSED_DUMMY_FOLDER_PATH_DIST}")
-
-set(COMPRESS_DUMMY_FILE_PATH_SRC "${FMIL_TEST_DIR}/try_to_compress_this_file.xml")
-set(COMPRESS_DUMMY_FOLDER_PATH_DIST "${TEST_OUTPUT_FOLDER}")
-set(COMPRESS_DUMMY_FILE_PATH_DIST "${TEST_OUTPUT_FOLDER}/successfully_compressed_this_file.zip")
-file(COPY "${COMPRESS_DUMMY_FILE_PATH_SRC}" DESTINATION "${COMPRESS_DUMMY_FOLDER_PATH_DIST}")
-
-#Create paths for the config_test.h
-if(WIN32)
-    set(DLL_OUTPUT_PATH "${TEST_OUTPUT_FOLDER}")
-
-    string(REPLACE "/" "\\\\" UNCOMPRESSED_DUMMY_FILE_PATH_SRC "${UNCOMPRESSED_DUMMY_FILE_PATH_SRC}")
-    string(REPLACE "/" "\\\\" UNCOMPRESSED_DUMMY_FILE_PATH_DIST "${UNCOMPRESSED_DUMMY_FILE_PATH_DIST}")
-    string(REPLACE "/" "\\\\" UNCOMPRESSED_DUMMY_FOLDER_PATH_DIST "${UNCOMPRESSED_DUMMY_FOLDER_PATH_DIST}")
-    string(REPLACE "/" "\\\\" COMPRESS_DUMMY_FILE_PATH_SRC "${COMPRESS_DUMMY_FILE_PATH_SRC}")
-    string(REPLACE "/" "\\\\" COMPRESS_DUMMY_FILE_PATH_DIST "${COMPRESS_DUMMY_FILE_PATH_DIST}")
-endif(WIN32)
```

Comment:
Moved all of it to the .c files. Really want to keep cmake code short and readable.